portfolio/src/main/webapp/style.css

```diff
@@ -11,4 +11,20 @@
 .vse-image {
   max-width: 100%;
   max-height: 100%;
-}
\ No newline at end of file
+}
+
+form {
```

Comment:
Would you mind including a screenshot of what the new UI looks like?

Reply:
Here you go:
![image](https://user-images.githubusercontent.com/66034575/84038187-9c7a1f80-a96d-11ea-80cb-acef3eb5ce59.png)